Example/Podfile.lock

```diff
@@ -1,5 +1,7 @@
 PODS:
   - AppSwizzle (1.3.1)
+  - HCaptcha (1.7.0):
+    - HCaptcha/Core (= 1.7.0)
```

Comment:
This is weird. (Note lines below.)

Reply:
Good point this is how pod install update it, and probably this leads to few extra warnings during Xcode build, I will doublecheck if possible to avoid this

Reply:
Thanks, concerned it could break validation in the future.

Reply:
If we declare `HCaptcha/Core` instead `HCaptcha` in Podfile, it looks simpler:
```ruby
...
- HCaptcha/Core (= 1.7.0)
...
```
instead
```ruby
...
- HCaptcha (1.7.0):
  - HCaptcha/Core (= 1.7.0)
...
```

I will update PR later tonight

Reply:
Updated